src/audio/asrc/asrc_config.h

```diff
@@ -16,7 +16,7 @@
 #if defined __XCC__
 /* For xt-xcc */
 #include <xtensa/config/core-isa.h>
-#if XCHAL_HAVE_HIFI3 == 1
+#if XCHAL_HAVE_HIFI3 == 1 || XCHAL_HAVE_HIFI4 == 1
```

Comment:
this isn't really a code reorganization... It's really not good practice to add such configuration changes in the middle or a code move...

This should be in a separate patch IMHO.